CMakeLists.txt

```
@@ -6,6 +6,11 @@ SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 # TODO: Is that actually used?
 INCLUDE(GenerateExportHeader)
 
+SET(WITH_GLOBE ON CACHE BOOL "Enable globe")
```

```suggestion
set(WITH_GLOBE ON CACHE BOOL "Enable globe")
```

I guess we'll do a batch update of style once the vcpkg stuff is settled